### PR TITLE
Officer add/remove functionality implemented

### PR DESCRIPTION
**Pull request by czhanacek**
Finally!

### CODE REVIEW

**Comment by t-walker** [comment]
@czhanacek it doesn't look like it's passing the build.

**Comment by czhanacek** [comment]
I know... When I merged I forgot to run the tests again. Fixing now.

File: app/controllers/officer_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  def index
+  end
+
+  def manage
+    @officers = Officer.order(:last)
+  end
+
+  def create_officer
+    officer = Officer.create(:first => params[:first], :last => params[:last], :email => params[:email], :phone => params[:phone ])
+    #rso = Rso.create({name: params[:name], nickname: params[:nickname]})
```
Comment (t-walker) [inline app/controllers/officer_controller.rb]: avoid leaving commented code in unless it's an example

File: app/controllers/rso_controller.rb [hunk 2]
```diff
@@ -32,8 +37,37 @@ def add_keyword
...
+      flash[:success] = "Keyword deleted successfully"
+    else
+      flash[:error] = "RSO not deleted"
+    end
+
+    redirect_to action: "manage"
+  end
+
+  def add_officer
+    newOfficer = Officer.find(params[:officer_id])
```
Comment (t-walker) [inline app/controllers/rso_controller.rb]: can this just be lower case officer like you have for rso?

File: app/controllers/rso_controller.rb [hunk 3]
```diff
@@ -32,8 +37,37 @@ def add_keyword
...
+      flash[:success] = "Officer added to RSO successfully"
+    rescue ActiveRecord::RecordInvalid => invalid
+      flash[:error] = "Officer not added to RSO: " + invalid.record.errors.full_messages.join(", ")
+    end
+
+
+    redirect_to action: "manage"
+  end
+
+  def delete_officer
```
Comment (t-walker) [inline app/controllers/rso_controller.rb]: this should be something more akin to removing an officer from an organization rather than deleting officer

Comment (czhanacek) [inline app/controllers/rso_controller.rb]: I thought "delete" removed the relationship between the two objects, whereas "destroy" removed the object itself.

**Pull request merged**